repos/system_upgrade/el7toel8/actors/opensshalgorithmscheck/tests/unit_test.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.actor import library
+from leapp.models import OpenSshConfig, OpenSshPermitRootLogin, Report
+from leapp.snactor.fixture import current_actor_context
+
+
+def test_no_config(current_actor_context):
+    with pytest.raises(StopActorExecutionError):
+        library.process(iter([]))
+
+
+osprl = OpenSshPermitRootLogin(value='no')
+
+
+@pytest.mark.parametrize('ciphers,expected_report', [
+    ('', False),
+    ('aes128-ctr', False),
+    ('aes128-ctr,aes192-ctr,aes256-ctr', False),
+    ('arcfour', True),
+    ('arcfour,arcfour128,arcfour256', True),
+    ('arcfour,aes128-ctr', True),
+    ('aes128-ctr,arcfour', True)])
+def test_ciphers(current_actor_context, ciphers, expected_report):
+    if ciphers:
```

Comment:
If you check for no_ciphers case by passing (None, False) instead of ('', False) as test data L26-L33 can be combined:
```current_actor_context.feed(OpenSshConfig(permit_root_login=[osprl], ciphers=ciphers))```

Reply:
Nice idea, thanks.